theme/accessibility.less

```diff
@@ -14,7 +14,8 @@
 }
 
 @media not all and (pointer:coarse) {
-    *:not([tabindex='-1']):focus {
+    *:not([tabindex='-1']):not(.blocklyGridPickerMenu):focus,
+    [role='menuitem']:focus {
```

Comment:
not just the blocklyGridpickerMenu but all blockly elements. Is there any way to exclude all elements under blocklySVG. We probably are going to want to exclude the monaco-editor too. Perhaps we can instead do this only on semantic ui classes.